src/market/market_utils.cairo

```diff
@@ -1318,14 +1323,20 @@ fn get_swap_impact_amount_with_cap(
     impact_amount
 }
 
+//Functions Added for libray#3 to compile
+
 /// Get the long and short open interest for a market based on the collateral token used.
 /// # Arguments
 /// * `data_store` - The data store to use.
 /// * `market` - The market to get the open interest for.
 /// * `collateral_token` - The collateral token to check.
 /// * `is_long` - Whether to get the long or short open interest.
 /// * `divisor` - The divisor to use for the open interest.
-fn get_open_interest(
+fn get_open_interest(data_store: IDataStoreDispatcher, market: Market, is_long: bool,) -> u128 {
+    0
+}
```

Comment:
shouldn't this be implemented?

Reply:
I think it's `get_open_interest_for_market_is_long` but it's this empty implementation that gets called so we should remove it